FormSchemas/jsonschema.json

```diff
@@ -122,6 +183,130 @@
       "items": {
         "type": "string"
       }
+    },
+    "item_assets": {
+      "type": "object",
+      "title": "Item Assets",
+      "properties": {
+      "cog_default": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "default":"image/tiff; application=geotiff; profile=cloud-optimized"
+          },
+          "roles": {
+            "type": "array",
+            "items": [
+              {
+                "type": "string"
+              }
+            ],
+            "default":[
+              "data",
+              "layer"
+            ]
+          },
+          "title": {
+            "type": "string",
+            "default":"Default COG Layer"
+          },
+          "description": {
+            "type": "string",
+            "default":"Cloud optimized default layer to display on map"
+          }
+        },
+        "required": [
+          "type",
+          "roles",
+          "title",
+          "description"
+        ]
+      }
+    },
+    "required": [
+      "cog_default"
+      ]
+    },
+    "renders": {
+      "type": "object",
+      "title": "Renders"
```

Comment:
looks like this is missing a type

Reply:
The renders object follows this schema: https://github.com/stac-extensions/render/blob/main/json-schema/schema.json

Do you think we should use that schema or just make it an object where someone can copy paste their renders object?

Reply:
without any properties defined, we will just get an empty fieldset from React-json-shema
<img width="1472" alt="Screenshot 2024-12-19 at 5 37 08 PM" src="https://github.com/user-attachments/assets/4da40302-0335-49a7-a708-325b633fa5fc" />

I think the library will support references within the json schema, so I can try copying the file to this repo, adding the reference, and styling it like the others.

Reply:
@smohiudd - It turns out that the react-jsonschema-form library "only supports local definition referencing". There is a separate [json-schema-ref-parser](https://github.com/APIDevTools/json-schema-ref-parser) library that does let us parse the json schema. I pulled that in and put a copy of the stac-extensions schema above in the public directory to try it out.  Unfortunately, it seems to be hanging up with some references.  I'm still trying to debug, but I pushed a commit with a partially working version.

Another work around would be something like allow some or all of the Renders object be a text area where users paste in the object.

Reply:
ok, I made that change and pushed it up.  I'm good with merging and trying it this way.

Reply:
Thanks!